crates/miden-lib/asm/miden/auth/rpo_falcon512.masm

```diff
@@ -104,7 +104,7 @@ export.verify_signatures.16
         sub.1 dup push.0.0.0 loc_load.PUB_KEY_MAP_IDX_LOC
         # => [owner_key_slot, [0, 0, 0, i-1], i-1, MSG]
 
-        exec.account::get_map_item
+        exec.account::get_map_item_init
```

Comment:
I would probably add a comment here explaining the reasoning for using `get_map_item_init`.